Controller/Admin/Order/OrderTabSepaMandate.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * Shop System Plugins:
+ * - Terms of Use can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/_TERMS_OF_USE
+ * - License can be found under:
+ * https://github.com/wirecard/oxid-ee/blob/master/LICENSE
+ */
+
+namespace Wirecard\Oxid\Controller\Admin\Order;
+
+use Wirecard\Oxid\Core\Helper;
+
+/**
+ * Controls the view for the order sepa mandate tab.
```

Comment:
Please change `sepa`/`Sepa` to uppercase `SEPA` for consistency (also, in another 7 occurrences).